src/plugins/robots/drone/simulator/pointmass3d_drone_model.cpp

```diff
@@ -58,11 +60,17 @@ namespace argos
       m_cAccelerationPrev.Set(0.0, 0.0, 0.0);
       /* reset the drone's previous angular acceleration */
       m_cAngularAccelerationPrev.Set(0.0, 0.0, 0.0);
-      /* TODO verify if the following variables are necessary */
+      /* reset the variables for PID controller  */
       m_cOrientationTargetPrev.Set(0.0, 0.0, 0.0);
       m_cAngularVelocityCumulativeError.Set(0.0, 0.0, 0.0);
       m_fAltitudeCumulativeError = 0.0;
       m_fTargetPositionZPrev = 0.0;
+      /* reset the gyro bias */   
+      m_fGyroBias = MEMS_GYRO_BIAS_INIT;
+      m_fAccelBias = MEMS_ACCEL_BIAS_INIT;   
+      m_fAngleRandomWalk = MEMS_GYRO_RANDOM_WALK_INIT;
```

Comment:
Perhaps this should be `m_fAngularVelocityRandomWalk`? or is it actually just an angle?

Reply:
There are two definitions for the random walk of the gyro. Some resources call it AngleRandomWalk, and some AngularRandomWalk but not AngularVelocityRandomWalk. 

It is really an angle. Since often, one is interested in using the rate sensor to track angle changes, not simply the angular rate. 

 I can change it to `m_fAngularRandomWalk` if it makes much more sense.

Reply:
I guess what seems a bit off here to me is that we have `m_fAngleRandomWalk` and `m_fVelocityRandomWalk`. I would expect both to be a position/angle or both to be the first derivative of that value.

If it is the case that only one is the first derivative, then what you have is already correct.